Iterable/iterable-web-sdk
# Add remaining events to web embedded messaging

## Description

- **Release Notes**
> Develop the remaining events for web embedded messaging.

> Update the corresponding test cases.

> We've also updated the sample app to showcase the tracking of embedded messaging events.

- **Test Steps**
> Start by launching the sample app, then select the 'embedded' option. Enter your user ID in the designated field. Modify the parameters you want to test in the code. Afterward, choose the option you wish to test and click. You will then be able to observe the different views.
